client/tools/spacewalk-koan/spacewalkkoan/spacewalkkoan.py

```diff
@@ -169,6 +169,7 @@ def initiate(kickstart_host, base, extra_append, static_device=None, system_reco
         k.virt_type           = None
         k.virt_bridge         = None
         k.no_gfx              = 0
+        k.gfx_type            = 'vnc'
```

Comment:
Can this create problems when older koan is used which does not know this parameter?
Maybe you need to do something similar as below with the "no_copy_default" option.

Reply:
Oh indeed. Fixed now